libr/core/cmd_anal.c

```diff
@@ -5904,27 +5904,35 @@ static char *getViewerPath() {
 }
 
 static char* graph_cmd(RCore *core, char *r2_cmd, const char *save_path) {
+	const char *quotes = "";
 	char *cmd = NULL;
 	const char *ext = r_config_get (core->config, "graph.gv.format");
 	char *dotPath = r_file_path ("dot");
-	if (!r_file_exists (dotPath)) {
+	if (!strcmp (dotPath, "dot")) {
 		free (dotPath);
 		dotPath = r_file_path ("xdot");
+		if (!strcmp (dotPath, "xdot")) {
+			free (dotPath);
+			return r_str_new ("agf");
+		}
+	}
+#if __WINDOWS__
+	if (dotPath[0] == '\"') {
+		quotes = "\"";
 	}
-	if (r_file_exists (dotPath)) {
-		if (save_path && *save_path) {
-			cmd = r_str_newf ("%s > a.dot;!%s -T%s -o%s a.dot;", r2_cmd, dotPath, ext, save_path);
+#endif
+	if (save_path && *save_path) {
+		cmd = r_str_newf ("%s > a.dot;!%s%s -T%s -o%s a.dot%s;",
```

Comment:
what about using ./ and then mv using the C api ? instead of escaping chars, checking boundaries, etc

Reply:
What do you mean under "./"?